account_reconcile_analytic_tag/models/account_bank_statement_line.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class AccountBankStatementLine(models.Model):
+    _inherit = "account.bank.statement.line"
+
+    analytic_tag_ids = fields.Many2many(
```

Comment:
Shouldn't it be this name following current fields nomenclature?

```suggestion
    manual_analytic_tag_ids = fields.Many2many(
```

Reply:
The analytical distribution field does not have the manual prefix but if the name is consistent, it is already changed.